addons/resource/models/resource.py

```diff
@@ -645,6 +645,13 @@ class ResourceCalendarAttendance(models.Model):
     hour_to = fields.Float(string='Work to', required=True)
     calendar_id = fields.Many2one("resource.calendar", string="Resource's Calendar", required=True, ondelete='cascade')
 
+    @api.constrains('hour_to', 'hour_from')
+    def _verify_hours(self):
```

Comment:
Please make it wkkr in batch.

Reply:
Better to use meaningful variable names.

``` python
        for attendance in self:
            if attendance.hour_from > 24 or attendance.hour_to > 24:
                raise ValidationError(_("Please enter hours between 1 to 24"))
            if (attendance.hour_to - attendance.hour_from) <= 0:
                raise ValidationError(_("Please use 24 hours format!"))
```